applications/TrilinosApplication/tests/test_neighbours.py

```diff
@@ -0,0 +1,99 @@
+﻿from __future__ import print_function, absolute_import, division
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics
+import KratosMultiphysics.mpi as KratosMPI
+import KratosMultiphysics.MetisApplication as KratosMetis
+import kratos_utilities
+
+
+def GetFilePath(fileName):
+    return os.path.dirname(os.path.realpath(__file__)) + "/" + fileName
+
+
+class TestMPICommunicator(KratosUnittest.TestCase):
+
+    def tearDown(self):
+        kratos_comm  = KratosMultiphysics.DataCommunicator.GetDefault()
+        rank = kratos_comm.Rank()
+        if rank == 0:
+            kratos_utilities.DeleteFileIfExisting("test_mpi_communicator.time")
+        kratos_utilities.DeleteFileIfExisting("test_mpi_communicator_"+str(rank)+".mdpa")
+        kratos_utilities.DeleteFileIfExisting("test_mpi_communicator_"+str(rank)+".time")
+        kratos_comm.Barrier()
+
+    def _ReadModelPart(self, input_filename, mp):

```

Comment:
please use the trilinos_import_model_part_utility here instead of doing the partitioning yourself
=> this utility will be moved to the MPI-core once that one has a nice py-module

Reply:
i just tried but for any reason it does not read correctly. I guess it is some sort of bad mixing of old and new data communicators, or maybe that the --using-mpi should be used.

btw this test should run the same with or without mpi. if you can suggest how to achieve that with the latest changes i would be very interested